src/Response.php

```diff
@@ -18,6 +19,11 @@ class Response
      */
     protected $content;
 
+    /**
+     * @var string|File|\SplFileInfo
```

Comment:
Just string to avoid sticking to Symfony here. In Router implementation (Silex) you can transform it in Symfony file if you want, but in fact it's not necessary.